badges/badges.md

```diff
@@ -1,26 +1,25 @@
-# Badges!
+# Badges
 
-### What are badges?
-The badges are a complement system to the Ladder - while ladder outlines career paths, the badges reward behaviors and achievements we respect and appreciate in our fellows.
+### What are the badges?
+The badges are a complementing tool to the Ladder - while ladder outlines career paths, badges reward behaviors and achievements we respect and appreciate in our team members.
 
 ### How can I create a badge?
-Simple. Badges must meet the following criteria:
-#### 1. A badge achievement must be a measurable, objective, pass fail criteria.
-###### Example Badge: *Good presenter*
-Do: Presented to an audience of at least 50 people
-Don't: Had a nice, well structured presentation
+Simply. Badges must meet the following criteria:
```

Comment:
"Simple" keeps an informal tone, and "Simply" can't be a sentence by itself. "Simple" stays

Reply:
fixed